Assets/SEETests/TestConfigIO.cs

```diff
@@ -800,6 +809,19 @@ private static void WipeOutSEECityAttributes(SEECity city)
             city.DataProvider = new PipelineGraphProvider();
         }
 
+        /// <summary>
+        /// Wipes out all attributes of <paramref name="markerAttributes"/>.
+        /// </summary>
+        /// <param name="markerAttributes">to be wiped out</param>
+        private static void WipeOutMarkerAttributes(MarkerAttributes markerAttributes)
+        {
+            markerAttributes.MarkerHeight++;
+            markerAttributes.MarkerWidth++;
```

Comment:
I don't quite understand why this incrementing is counted as "wiping out". If this is intended, maybe a short explanatory comment would help.

Reply:
"Wiping out" means in those cases just that a value different from the default or from a value previously set value is assigned so that we could notice any difference between the "wiped out" and loaded values. That holds true for all "wiping".

I could put this in a general comment. It is not specific to this particular assignment though.